implementations/micrometer-registry-statsd/src/test/java/io/micrometer/statsd/StatsdMeterRegistryTest.java

```diff
@@ -455,7 +455,7 @@ public boolean enabled() {
             .build();
 
         registry.counter("some.metric").increment();
-        assertThat(registry.queueSize()).as("counter increment should already be processed").isZero();
+//        assertThat(registry.queueSize()).as("counter increment should already be processed").isZero();
```

Comment:
Is it possible to check some way that the processor has already processed the item generated by the previous line?

Reply:
Or that it has no subscribers, which is what we expect here.

Reply:
yes for the latest: the `DirectProcessor` is a `Scannable`. You can thus get a stream of its subscribers and count: `processor.inners().count()`.

Note that `inners()` returns a `Stream<Scannable>` that reflects the correct size, but each element is a wrapper around the actual `Subscriber`(s) you connected to the processor. You can even access the actual subscribers that can be introspected by doing:

```java
processor.inners() //Stream<Scannable>
    .map(inner -> inner.scan(Attr.ACTUAL))
    .filter(Objects::nonNull); //Stream<Scannable> of introspectable subscribers
```